x-pack/plugins/security_solution/public/common/containers/matrix_histogram/index.test.ts

```diff
@@ -26,7 +27,9 @@ describe('useMatrixHistogram', () => {
 
   it('should update request when props has changed', async () => {
     const localProps = { ...props };
-    const { rerender } = renderHook(() => useMatrixHistogram(localProps));
+    const { rerender } = renderHook(() => useMatrixHistogram(localProps), {
+      wrapper: TestProviders,
+    });
```

Comment:
Note for posterity: as discussed, this is a fix needed as result of the use of experimental features (stored in redux).